README.md

```diff
@@ -10,26 +10,39 @@ The documentation is built using Markdown pages, which can be converted into a n
 
 ## Build dependencies
 
-The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.76.0 and later. See `.tool-versions` for the currently used version
+The documentation site uses [Hugo](https://gohugo.io/) including the extended version of Hugo. Version v0.122 is used per (see #https://github.com/medic/cht-docs/issues/1302) as denoted in `.tool-versions` [file](https://github.com/medic/cht-docs/blob/main/.tool-versions).
```

Comment:
```suggestion
The documentation site uses the extended version of [Hugo](https://gohugo.io/). See the `.tool-versions` [file](https://github.com/medic/cht-docs/blob/main/.tool-versions) for the latest version of Hugo supported for these docs.
```

Hard-coding the actual version value here gives us another place to have to update when the supported version changes.  Seems better to just point folks to the `.tool-versions` file as the source of truth.

Reply:
natch 0a832ef1